Proposals/BTC Forks Roadmap Proposal.md

```diff
@@ -85,91 +48,78 @@ A decentralised system works best when all its components are in equilibrium.
 
-[http://www.bitcoinwednesday.com/wp-content/uploads/2016/02/Bitcoins-Incentive-Structure.pdf](http://www.bitcoinwednesday.com/wp-content/uploads/2016/02/Bitcoins-Incentive-Structure.pdf)
-
----
-
-**5.ii. Bitcoin Evolutionary Processes (BEP)**
-
-Turning bitcoin development into an evolutionary system where progress is market driven rather than driven by a select few ‘official’ developers. The bitcoin evolutionary process is made up of three parts. The first part is the **Consensus Abstraction Layers**. The second part is the **Accelerated Concept Development**. The third part is the **Implementation Readiness Levels**. Each part is designed to solve a different problem in bitcoin currently.
+**Our Options**
 
+While there are many different proposals on how to solve this problem, most solutions boil down to one of two solutions. One solution is to change the economic model of bitcoin so that the bitcoin network does not scale much further on-chain, but rather exra layers are developed on top of the bitcoin network. These extra layers would connect to bitcoin for security but would not have the same security model of bitcoin. The Lightning is an example of a layer 2 network. This solution could be boiled down to "Bitcoin: An Electronic Settlement Network". Another solution is to allow the bitcoin network to scale on-chain by allowing more transactions to happen directly on the bitcoin network. This solution would keep the economic model of bitcoin that has worked over the past 7 years and would extend it to the future. "This solution can be boiled down to "Bitcoin: A Peer-to-Peer Electronic Cash Network for Payments and Settlement Transactions"
 
-**5.iii. Consensus Abstraction Layers (CAL)**
+Currently, the development of the bitcoin reference client is done by the group of bitcoin developers called "Core". While there are various other smaller development teams that offer their own bitcoin clients, the Core developers control the development of the client that most network participants are aware of. The Core developers of bitcoin have decided to use the solution of "Bitcoin: An Electronic Settlement Network". A significant (although not necessarily a majority) number of the network participants do not feel that this solution is the best way forward.
```

Comment:
>the group of bitcoin developers called "Core"

I would say 'calling themselves "Core"' - they are free to rebrand at any time, after all.